Google.PowerShell/BigQuery/GcbqProject.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Google.Apis.Bigquery.v2;
+using Google.Apis.Bigquery.v2.Data;
+using System.Management.Automation;
+using System;
+
+namespace Google.PowerShell.BigQuery
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Lists all projects to which you have been granted any project role.
+    /// </para>
+    /// <para type="description">
+    /// Queries to find which projects you have been granted any project role in and returns them in the form of ProjectsData objects.
+    /// The flag -MaxResults can be used to specify the maximum number of results retrieved at once, and -PageToken can be used to
+    /// specify an offset into the project list at which to start.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> Get-GcbqProjects -MaxResults 20</code>
+    ///   <para>This command will list up to the first 20 BigQuery projects that you've been granted any roles for.</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Get-GcbqProjects -MaxResults 20 -PageToken 20</code>
+    ///   <para>This command will list the second page of up to 20 BigQuery projects that you've been granted any roles for.</para>
+    /// </example>
+    /// <para type="link" uri="(https://cloud.google.com/bigquery/docs/reference/rest/v2/projects)">
+    /// [BigQuery Projects]
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "GcbqProject")]
+    public class GetGcbqProject : GcbqCmdlet
+    {
+        /// <summary>
+        /// <para type="description">
+        /// The maximum number of results to return.
+        /// </para>
+        /// </summary>
+        [Alias("mr")]
+        [Parameter(Mandatory = false)]
+        public uint? MaxResults { get; set; }
```

Comment:
Don't use this. As written, it will only make the request page size smaller, requiring more requests but iterating all the projects nevertheless. In the rare case that a user does want to limit the number of returned projects, they can pipe this Cmdlet to `Select-Object` like so:
```
Get-GcbqProject | Select-Object -First 100
```
That will cause `WriteObject` to throw an exception once 100 projects have been returned, short circuiting further requests. In an interactive environment, `Ctrl-C` will set `Stopping` to `true`.